test/backup.ts

```diff
@@ -474,8 +577,8 @@ describe('Bigtable/Backup', () => {
         callback(null, ...args);
       };
 
-      backup.restore(
-        tableId,
+      backup.restoreTo(
```

Comment:
I think we may want to leave some of these tests as is, so we don't lose coverage on `.restore()` functionality?

Reply:
To have gRPC call implementation only in one place `Backup#restore()` (as more restrictive wrapper since can only restore to the same instance) [delegates](https://github.com/googleapis/nodejs-bigtable/blob/53d9ca586c1a86aa6f86c571dc918ea49dcb85a2/src/backup.ts#L507-L514) to `Backup#restoreTo()` and passing `this.cluster.instance` value as the [`config.instance`](https://github.com/googleapis/nodejs-bigtable/blob/53d9ca586c1a86aa6f86c571dc918ea49dcb85a2/src/backup.ts#L529) param.
The current unit tests for `Backup#restore()` are to verify that it properly delegates to `Backup#restoreTo()` with and without optional `gaxOptions` param.

Reply:
got it, just want to confirm we still have at least one `restore` test in place for coverage?